scaleway/resource_rdb_instance.go

```diff
@@ -114,6 +118,29 @@ func resourceScalewayRdbInstance() *schema.Resource {
 				Computed:    true,
 				Description: "Volume size (in GB) when volume_type is not lssd",
 			},
+			"private_network": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "List of private network to expose your database instance",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ip": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.IsCIDR,
+							Description:  "The IP of your service",
```

Comment:
What if we have a DHCP configuration happening in the private network. Range wise, is there tests on rdb side to verify that the IP range we ask is contained in the one allowed by the private network?

Reply:
A test that tests how I can combine DHCP and IP settings should be added to test that it works. Adding it to the documentation should be useful too as it presents an interesting use case.